SUMMARY.md

```diff
@@ -63,8 +67,19 @@
     - [Interface and Registers](./hw/ip/otbn/data/otbn.hjson)
     - [Checklist](./hw/ip/otbn/doc/checklist.md)
 
-- [Hardware IP Blocks](./hw/ip/README.md)
-  - [Analog to Digital Converter Control](./hw/ip/adc_ctrl/README.md)
+- [HWIP Blocks](./hw/ip/README.md)
```

Comment:
FWIW I prefer hardware IP/blocks in documentation

Reply:
I quite like the acronym HWIP because it is similar to VIP (Verification IP) which is an industry-standard term. However it probably should be avoided at the top-level to prevent jargon without definition. I'll change to "Hardware Blocks" in the SUMMARY.md